docs/en/reference/schema-representation.rst

```diff
@@ -76,6 +76,34 @@ All schema assets reside in the ``Doctrine\DBAL\Schema`` namespace.
 
     This chapter is far from being completely documented.
 
+Table
+~~~~~~
+
+Represents a table in the schema.
+
+Vendor specific options
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The following options, that can be set using `default_table_options`, are completely vendor specific
+and absolutely not portable.
+
+-  **charset** (string): The character set to use for the table. Currently only supported
+  on MySQL.
+
+-  **engine** (string): The DB engine used for the table. Currently only supported on MySQL.
+
+-  **unlogged** (boolean): Set a PostgreSQL table type as
+  `unlogged <https://www.postgresql.org/docs/current/sql-createtable.htmll>`_
+
+You can set this options via doctrine configuration. For example:
+
+.. code-block:: yaml
+
+    doctrine:
+        dbal:
+            default_table_options:
+                unlogged: true
```

Comment:
This block does not belong here because that configuration format is specific to [DoctrineBundle](https://github.com/doctrine/DoctrineBundle).